doc/rst/tools/protoc-gen-chpl/protoc-gen-chpl.rst

```diff
@@ -0,0 +1,186 @@
+===============
+Chapel Protobuf
+===============
+
+Chapel Protobuf is a Google Protocol Buffers implementation for Chapel.
+`Protocol Buffers`_ is a language-neutral, platform-neutral, extensible mechanism
+for serializing structured data. The protocol buffer language is a language for 
+specifying the schema for structured data. This schema is compiled into language
+specific bindings.
+
+This project is comprised of two components:
+
+* Code generator: The ``protoc-gen-chpl`` tool is a compiler plugin to ``protoc``, the protocol
+  buffer compiler. It provides arguments to the ``protoc`` compiler so that it knows how to
```

Comment:
"It provides arguments to the ``protoc`` compiler so that it knows how to generate Chapel specific code for a given ``.proto`` file."

-> 

"It enables the ``protoc`` compiler to generate Chapel code."

(You cover the argument part later).